tests/utils.go

```diff
@@ -4281,6 +4288,16 @@ func HasCDI() bool {
 	return HasDataVolumeCRD()
 }
 
+func isARM64() bool {
+	return rt.GOARCH == "arm64"
+}
+
+func SkipIfARM64() {
+	if isARM64() {
+		Skip("Skip test on arm64, arm64 does not support cpu model")
```

Comment:
Oh, I meant changing `SkipIfARM64()` to `SkipIfARM64(reason string)` and then call it like this when skipping:

```
SkipIfARM64("arm64 does not support setting CPU models")

Reply:
Updated